proguard/proguard-square-okio.pro

```diff
@@ -0,0 +1,5 @@
+# Okio
+-keep class sun.misc.Unsafe { *; }
+-dontwarn java.nio.file.*
+-dontwarn org.codehaus.mojo.animal_sniffer.IgnoreJRERequirement
+-dontwarn okio.**
```

Comment:
no rules for okio (included in okhttp)
https://github.com/square/okio#proguard

Reply:
I can't compile if I remove the line `-dontwarn org.codehaus.mojo.animal_sniffer.IgnoreJRERequirement` 🙀  

Do you know a way to fix it?

Reply:
```
-dontnote retrofit2.Platform
-dontwarn retrofit2.Platform$Java8
```

can compile if use above rule in retrofit proguard rule(and it's ok to remove okio proguard rule)

cuz `org.codehaus.mojo.animal_sniffer.IgnoreJRERequirement` is referenced from `retrofit2.Platform$Java8` class.